test/utils/db.factories.ts

```diff
@@ -0,0 +1,73 @@
+import { Customer } from 'src/customers/domain/customer.entity';
+import { Location } from 'src/products/domain/location.entity';
+import { Order } from 'src/orders/domain/order.entity';
+import { OrderDetail } from 'src/orders/domain/order-detail.entity';
+import { Product } from 'src/products/domain/product.entity';
+import { ProductCategory } from 'src/products/domain/product-category.entity';
+import { Stock } from 'src/products/domain/stock.entity';
+
+export const createCustomer = (): Customer => {
```

Comment:
**suggestion:** you should try to move this method factories for the constants somewhere in a shared place, so you can use it for the unit tests as well